includes/Story_Post_Type.php

```diff
@@ -318,6 +325,10 @@ public static function admin_enqueue_scripts( $hook ) {
 						'statuses' => '/wp/v2/statuses',
 						'fonts'    => '/amp/v1/fonts',
 					],
+					'metadata'         => [
+						'publisher' => self::get_publisher_data(),
+						'poster'    => plugins_url( 'assets/images/fallback-poster.jpg', WEBSTORIES_PLUGIN_FILE ),
```

Comment:
Should the poster always load the default here? How is this set by the user?

Reply:
This is just the fallback poster. Perhaps I should rename it.

The actual poster image is derived from the featured image. See our discussion above: https://github.com/google/web-stories-wp/pull/346#discussion_r385721044

Reply:
Changing the name here, would make this clearer.